Kajabi/sage-lib
# chore(dynamic-select): adds deprecation messaging

## Description
Due to vite work in the monolith and issues being cause by the dynamic select component, it has been removed from KP. Since the component was built for KP specifically a deprecation message will be added to the component page.


## Solution

Adds deprecation messaging and updates status in helper.

|  Screenshot  |
|--------|
|![Screenshot 2024-04-05 at 2 29 44 PM](https://github.com/Kajabi/sage-lib/assets/1175111/4de3f00b-9f6e-4f06-a550-a94e024cb53c)|


## Testing in `sage-lib`

- Navigate to [dynamic select](http://localhost:4000/pages/component/dynamic_select?tab=preview)
- Verify deprecation message appears


## Testing in `kajabi-products`
1. (**LOW**) Adds deprecation message for dynamic select. No effect on KP.


## Related
https://kajabi.atlassian.net/browse/DSS-551
